cpp/ah_wrapper/src/wrapper.cpp

```diff
@@ -77,4 +82,54 @@ int AHWrapper::read_write_once(const std::array<float, 6> &cmd_values,
   }
 
   return 0;
+}
+int AHWrapper::write_once(const std::array<float, 6> &cmd_values,
+      Command cmd,
+      uint8_t reply_mode)
+{
+  switch (cmd) {
+    case POSITION: m_buffer_idx = build_pos_msg(cmd_values, m_buffer, hand.address, reply_mode); break;
+    case VELOCITY: m_buffer_idx = build_vel_msg(cmd_values, m_buffer, hand.address, reply_mode); break;
+    case CURRENT:  m_buffer_idx = build_tor_msg(cmd_values, m_buffer, hand.address, reply_mode); break;
+    case DUTY:     m_buffer_idx = build_dut_msg(cmd_values, m_buffer, hand.address, reply_mode); break;
+  }
+  m_stuffed_idx = ppp_stuff(
+      m_buffer.data(), m_buffer_idx,
+      m_stuffed_buffer.data(), STUFFED_BUFFER_SIZE);
```

Comment:
Could you change the formatting here so it's the same as in the `read_write_once` function?
```suggestion
  m_stuffed_idx = ppp_stuff(m_buffer.data(), m_buffer_idx,
                            m_stuffed_buffer.data(), STUFFED_BUFFER_SIZE);
```